packages/core/src/navigation/navigation-start.element.scss

```diff
@@ -10,14 +10,17 @@
 :host {
   --color: inherit;
   --line-height: inherit;
+  --focus-width: #{$cds-alias-object-border-width-300};
   --font-size: inherit;
   --font-weight: inherit;
   --text-transform: capitalize;
+  padding-left: var(--focus-width);
 }
 
 :host(:hover),
 :host([_has-focus]) {
   background: #{$cds-alias-object-interaction-background-hover};
+  border: 1px solid var(--cds-alias-object-interaction-background-highlight);
```

Comment:
`1px` here too

Reply:
Updated to `var(--cds-alias-object-border-width-100)`